src/transformers/models/pix2struct/configuration_pix2struct.py

```diff
@@ -357,9 +357,10 @@ def __init__(
         initializer_factor=1.0,
         initializer_range=0.02,
         is_vqa=False,
+        tie_word_embeddings=False,
```

Comment:
@younesbelkada for confirmation.

Pix2Struct has Vision, Text, and Global model (and therefore config).

The text properly sets the tie_word_embeddings to False, but the global one didn't and therefore the global model would forcefully set the word embeddings to tied even when it shouldn't.

I think this is also independant from this safetensors PR.